apps/backend/src/libs/modules/server-application/libs/enums/enums.ts

```diff
@@ -0,0 +1,2 @@
+export { APIPath } from "~/libs/enums/enums.js";
+export { AuthApiPath } from "~/modules/auth/libs/enums/enums.js";
```

Comment:
Shouldn't be exported from here. Auth files should only be exported from auth module. And imported from auth module as well